samples/Web/deploy-webapp-ip-restrictions/azurepolicy.json

```diff
@@ -0,0 +1,75 @@
+{
+    "properties": {
+        "displayName": "Web App default IP restrictions",
+        "mode": "indexed",
+        "description": "Configures Web App IP restriction to allow access from specified public IP addresses",
```

Comment:
>specified public IP addresses [](start = 79, length = 29)

The user should be able to provide these IP addresses as parameters to the policy.  Otherwise, it becomes far too specific.

Reply:
I would love to parameterize the IP addresses.  If anyone has a suggestion on how to accomplish that for this specific case (an array of objects) in Azure Policy JSON, I'm happy to learn more.